src/assets/wallets.md

```diff
@@ -99,29 +99,92 @@ curl -X GET "https://service.centrapay.com/api/wallets/WRhAxxWpTKb5U7pXyxQjjY/tr
 |:-----------------|:--------------------------------------------------------------------------|
 | amount           | Absolute transaction amount in cents.                                     |
 | value            | Change to the wallet's balance in cents when the transaction was applied. |
-| currency         | The currency of the transacion (same for all transactions in the wallet). |
 | createdAt        | Transaction timestamp as ISO8601 date string.                             |
 | type             | Transcation type: "transfer", "deposit" or "withdrawal".                  |
 | destWalletId     | Id of the destination wallet if applicable.                               |
 | srcWalletId      | Id of the source wallet if applicable.                                    |
-| paymentRequestId | Id of the related payment request                                         |
-| paymentTxnId     | Id of the related payment request transaction.                            |
-| depositId        | Id of the related deposit request.                                        |
-| withdrawalId     | Id of the related withdrawal request.                                     |
+| destParty        | Display value for party receiving funds if applicable. See note below.    |
+| srcParty         | Display value for party providing funds if applicable. See note below.    |
+| paymentRequestId | Id of the related payment request if applicable.                          |
+| withdrawalId     | Id of the related withdrawal request if applicable.                       |
+| topupId          | Id of the related topup funds transfer request if applicable.             |
+| assetTransferId  | Id of the related asset transfer request if applicable.                   |
+
+**Party Values**
+
+The "destParty" and "srcParty" are optionally provided depending on the purpose
+of the transaction. Values may include a merchant name, user handle, masked user
+email, masked user phone, or bank account.
 
-**Example response payload**
+
+**Example response payload (Payment)**
 
 ```json
 {
   "items": [
     {
       "amount": "350",
       "value": "-350",
-      "currency": "NZD",
       "createdAt": "2020-06-17T18:00:23.000Z",
       "type": "transfer",
-      "destWalletId": "EfYJd5tZQ63CrehgTP2RPB",
-      "srcWalletId": "EBVSreNmpsE2Pazw3SipXC"
+      "destParty": "Centrapay Cafe",
+      "destWalletId": "GfYJd5tZQ63CrehgTP2RPB",
+      "srcWalletId": "EBVSreNmpsE2Pazw3SipXC",
+      "paymentRequestId": "76961bc8-878b-11eb-bc82-abcc0f5e1f60"
+    }
+  ]
+}
+```
+
+**Example response payload (Topup)**
+
+```json
+{
+  "items": [
+    {
+      "amount": "5000",
+      "value": "5000",
+      "createdAt": "2020-06-17T18:00:23.000Z",
+      "type": "deposit",
+      "srcParty": "12-4000-100001-00",
```

Comment:
Would this be PII that we need to blank out? Something like `"12-****-******-00"`